packages/api-explorer/e2e/e2e.spec.ts

```diff
@@ -255,7 +255,9 @@ describe('API Explorer', () => {
       await expect(page).toMatchElement('button', { text: 'Types (0)' })
       await expect(page).toClick('a', { text: 'Get Workspace' })
       await expect(page).toMatchElement('h2', { text: 'Get Workspace' })
-      await expect(page.url()).toEqual(`${v40}/methods/Workspace/workspace`)
+      await expect(page.url()).toEqual(
+        `${v40}/methods/Workspace/workspace?sdk=kt&s=get+workspace`
```

Comment:
seems like we have some state being carried over between tests. Let's clear local storage between tests using the `beforeEach` api.